config/credhub_secret_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("CredhubSecret", func() {
 			Expect(fakeAuthClient.DoAuthenticatedPutCallCount()).Should(Equal(1))
 
 			url, bodyType, body, token := fakeAuthClient.DoAuthenticatedPutArgsForCall(0)
-			Expect(url).To(Equal(secretsURI + "/application/cloud/one"))
+			Expect(url).To(Equal(secretsURI + "/application/cloud/master/one"))
```

Comment:
This doesn't exercise all the characters that we say are valid in the regex - I think it should.